source/_components/alarm_control_panel.group.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Alarm Group"
+description: "Instructions how to setup the alarm group platform."
+date: 2017-11-22 00:12
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Alarm
+ha_release: 0.59
+---
+
+The `group` alarm platform allows you to combine multiple `alarm` platforms into a single service.
+
+To use this notification platform in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+alarm:
+  - name: ALARM_NAME
+    platform: group
+    code_format: '\d+'
+    panels:
+    - panel: alarm_control_panel.garage
+    - panel: alarm_control_panel.living_room
+    - panel: alarm_control_panel.bedroom
+    - panel: alarm_control_panel.tampering
+```
+
+Configuration variables:
+
+{% configuration %}
+  name:
+    description: The name of the group
+    required: true
```

Comment:
According to [the code](https://github.com/home-assistant/home-assistant/pull/11323/files#diff-e9227afef3b89ba1d9c7d8dd38984134R36) this is not required.